src/root/components/ListView/Message/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import BlockLoading from '#components/block-loading';
+import styles from './styles.module.scss';
+
+export interface Props {
+  className?: string;
+  empty?: boolean;
+  pending?: boolean;
+  errored?: boolean;
+  message?: React.ReactNode;
+  pendingMessage?: React.ReactNode;
+  emptyMessage?: React.ReactNode;
+  erroredMessage?: React.ReactNode;
+}
+
+function Message(props: Props) {
+  const {
+    className,
+    empty,
+    pending,
+    errored,
+    message: messageFromProps,
+    emptyMessage = 'No data available',
+    erroredMessage = 'Oops! We ran into an issue',
+  } = props;
+  console.log(props);
```

Comment:
Please remove such console.log after using it. I think "yarn typecheck" also mentions such console statements.